scripts/generate-package

```diff
@@ -14,6 +14,10 @@ const packages = [
     manifest: {
       name: '@cloudscape-design/theming-build',
       main: './build/index.js',
+      exports: {
+        '.': './build/index.js',
+        './internal': './build/internal.js'
```

Comment:
Moving forward, we should replace all current imports

```js
import { buildThemedComponentsInternal } from "@cloudscape-design/theming-build";
```

to the new subpath

```js
import { buildThemedComponentsInternal } from "@cloudscape-design/theming-build/internal";
```

Follow our standard naming convention where all team-internal APIs are coming from `/internal` imports